frontend/common/src/components/inputPartials/InputLabel/InputLabel.tsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable jsx-a11y/label-has-for */
```

Comment:
If you want to zap out the entire file, please add a new issue to the board and note the issue number here so we can come back and give it another try sometime.

Reply:
@esizer 
I think linting was upset because `label` couldn't be conveniently wrapped around `input`
From which I thought this would be a complex fix with needed refactoring, do you want to weigh in on whether it would be a complex fix or not? Feel free to check out the latest commit if you want to poke around

Reply:
Yeah, so this is a strange thing that most linters cannot handle. Since the label and input are in different files the linter cannot determine if they are associated with one another using `htmlFor`.

It will take a bit of extra work, but you can make the rule aware of these components.

Ref: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/label-has-associated-control.md

```json
{
  "rules": {
    "jsx-a11y/label-has-associated-control": [ 2, {
      "labelComponents": ["CustomInputLabel"],
      "labelAttributes": ["label"],
      "controlComponents": ["CustomInput"],
      "depth": 3,
    }],
  }
}
```